pydoctor/templates/common.html

```diff
@@ -5,51 +5,75 @@
   <head>
     <title><t:slot name="title">Something</t:slot> : API documentation</title>
     <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
-    <link rel="stylesheet" type="text/css" href="apidocs.css" />
-    <script t:render="ifusesorttable" type="text/javascript"
-            src="sorttable.js"></script>
-    <script t:render="pydoctorjs" type="text/javascript"
-            src="pydoctor.js"></script>
+
+    <link rel="stylesheet" type="text/css" href="bootstrap.min.css" />
+    <link rel="stylesheet" type="text/css" href="apidocs.css"/>
+
   </head>
   <body>
-    <t:slot name="heading"><h1>Heading</h1></t:slot>
-    <p>
-      <span id="part"><t:slot name="part">Part of something</t:slot></span>
-      <a t:render="source">View Source</a>
-      <a t:render="inhierarchy">View In Hierarchy</a>
-    </p>
-    <div>
-      <t:slot name="extras">
-        A docstring.
-      </t:slot>
-    </div>
-    <div class="docstring">
-      <t:slot name="docstring">
-        A docstring.
-      </t:slot>
-    </div>
 
-    <p t:render="splittingLinks">
-      <a id="showSplitLink" href="#" onclick="showSplit()" class="jslink">Split Table into Classes</a>
-      <a id="showBigLink" href="#" onclick="showBig()" class="jslink">Show Methods in One Table</a>
-    </p>
-    <div id="splitTables">
-      <t:slot name="mainTable" />
-      <t:invisible t:render="baseTables">
-        <p>
-          Inherited from <t:slot name="baseName" />:
-        </p>
-        <t:slot name="baseTable" />
-      </t:invisible>
-    </div>
-    <div id="bigTable" t:render="bigTable">
-    </div>
-    <t:slot name="packageInitTable" />
+    <nav class="navbar navbar-default">
```

Comment:
And finally, all these navbar blocks are the same aren't they? Is there really no way to combine them?